login-workflow/src/screens/Login.tsx

```diff
@@ -31,18 +31,24 @@ import clsx from 'clsx';
 
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
-        formFields: {
-            marginBottom: theme.spacing(3) + 22, // height of error message
+        emailFormField: {
+            marginBottom: theme.spacing(5),
```

Comment:
what was the reason for taking out the 22 here?

Reply:
I was doing general cleanup on spacing while doing the mobile responsiveness story. This was a result of that. I can check this again to be sure, but the new spacing should accommodate for the error text appropriately.

Reply:
So the hardcoded 22 is in here to prevent the form fields from moving when the error condition appears (if we don't do this, then the password field will shift down on error, which we didn't want). So we add the height of the error message to the margin when there is no error and remove it when there is an error.

It should be theme.spacing(4) + 22 on desktop and theme.spacin(3) + 22 on mobile though.